data/repositories/fedora-42.json

```diff
@@ -60,6 +60,12 @@
     }
   ],
   "riscv64": [
+    {
+      "name": "fedora",
+      "baseurl": "http://fedora.riscv.rocks/repos-dist/f42/latest/riscv64",
```

Comment:
This one gives me a 404 :thinking:

Reply:
Yeah, this one is "aspirational", i.e. I expect it to appear soon(ish) but I can remove it again and only add it once the f42 repos appear.